hspy_bundle/configure_installer.py

```diff
@@ -161,10 +168,8 @@ def create_installers(self):
                         fa.write(line.replace("__NSIS_PLUGINS__",
                                               get_nsis_plugins_path()))
                     elif "__HSPY_ICON__" in line:
-                        icons = self.get_full_paths(
-                            "python-*\\Lib\\site-packages\\hyperspy\\data\\"
-                            "hyperspy_bundle_installer.ico", a)
-                        fa.write(line.replace("__HSPY_ICON__", icons))
+                        fa.write(line.replace("__HSPY_ICON__",
+                                              get_icon_path()))
```

Comment:
We could, but we could also get rid of `copy_files` and insert the path of the files that we currently copy in the NSIS script. The latter is actually cleaner as `copy_files` leaves those useless files in the directory in which we create the bundle. However, I suggest that we leave things as they are: if it isn't broken don't touch it ;)